lib/core/common/helper/env_helper.dart

```diff
@@ -14,16 +12,8 @@ Future<Map<String, String>> loadEnvs(
   // ignore: unawaited_futures
   runZonedGuarded(
     () async {
```

Comment:
There's some code that you don't need anymore, for example, `runZonedGuarded`. That code was used to handle the exception, but now with the optional parameter ju can just remove it